New_File2

```diff
@@ -0,0 +1,82 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+from utils.constants import (FB_URL, FIRST_NAME, SUR_NAME, EMAIL_ADDRESS, NEW_PASSWORD)
+
+class SignUpForm():
+     """This class contains functions"""
```

Comment:
@ruttaba Not a meaningful docstring. Almost all classes are meant to contain functions.